airbyte-integrations/template/python-source/README.md

```diff
@@ -1,3 +1,32 @@
+### Create a Python Airbyte Source
+1. If you are trying to create an Airbyte Source from a Singer Tap, you should follow [this guide](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/template/singer-source/README.md) instead.
+1. Copy the template into a directory with your integration name. `cp -r airbyte-integrations/template/python-source airbyte-integrations/source-SOURCENAME`.
+1. ```
+   mv airbyte-integrations/source-SOURCENAME/template_python_source airbyte-integrations/source-SOURCENAME/SOURCENAME_source
```

Comment:
Can you update to follow the new directory structure?

Reply:
discussed offline, will need to happen as part of https://github.com/airbytehq/airbyte/pull/607 when it's merged to master.